src/app/extensions/evaluators/navigation.evaluators.ts

```diff
@@ -75,7 +75,7 @@ export function isLibraries(
   ...args: RuleParameter[]
 ): boolean {
   const { url } = context.navigation;
-  return url && url.endsWith('/libraries');
```

Comment:
How that behave when drilling down the files in the Library? previously this one was checking just the root one (ends with) but not the folders inside

Reply:
was a bug in my previous commit.